ui/apps/platform/src/Containers/AppPage.tsx

```diff
@@ -20,14 +20,21 @@ function AppPage(): ReactElement {
             <AppPageTitle />
             <AppPageFavicon />
             <Switch>
-                <Route path={loginPath} component={LoginPage} />
-                <Route
-                    path={authorizeRoxctlPath}
-                    render={(props) => <LoginPage {...props} authorizeRoxctlMode />}
```

Comment:
For my own understanding - the `...props` here are just the match/history/location props provided by react-router right? We don't lose anything by not forwarding these props in the change?

Reply:
I _believe_ so. The route props are also passed regardless here but I'm guessing the original author wanted to be more explicit. Although neither location, match, or history is being used in the `LoginPage` component, instead `window.location` is being used :thinking:

Here's at least the link to [render props documentation](https://v5.reactrouter.com/web/api/Route/render-func)

Reply:
@bradr5 did TypeScript require `authProviders={undefined}` prop here?

Does explicit prop conflict with access to Redux store here:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Login/LoginPage.js#L309

Reply:
TypeScript was requiring `authProviders` be set here since it's required in `LoadingOrForm` inside of `LoginPage.js`. However, regardless of whats being passed as a prop, the redux store `selectors.getLoginAuthProviders` is overriding whatever prop is being passed. Before the changes I made, it appears that `undefined` was being passed but typescript wasn't catching that as an issue since we were using the `Route` -> `component` prop. That's one of the benefits of making the incremental change before fully upgrading to v6.

It does seem like @dvail suggestion to default auth providers to an empty array makes the most sense. `undefined` would actually break the component if it wern't for redux and `mapStateToProps`.

Reply:
Yes, super. I agree with you and David. Thank you for investigating.